README.md

```diff
@@ -189,56 +189,244 @@ Minimal required IAM permissions:
 }
 ```
 
+For convenience, you can download it using:
+
+```bash
+curl \
+--fail \
+--silent \
+--write-out "Reponse code: %{response_code}\n" \
+https://raw.githubusercontent.com/qonto/prometheus-rds-exporter/main/configs/aws/policy.json \
+-o /tmp/prometheus-rds-exporter.policy.json
+```
+
 ## Installation
 
-### Locally
+See [Development environment](#development-environment) to start the Prometheus RDS exporter, Prometheus, Grafana with dashboards in a minute.
 
-1. Connect on AWS with any method
+### AWS EKS
+
+**Recommanded method** to deploy on AWS EKS using [IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) and Helm.
+
+1. Create an IAM policy
 
     ```bash
-    aws configure
+    IAM_POLICY_NAME=prometheus-rds-exporter
+
+    # Download policy payload
+    curl --fail --silent --write-out "Reponse code: %{response_code}\n" https://raw.githubusercontent.com/qonto/prometheus-rds-exporter/main/configs/aws/policy.json -o /tmp/prometheus-rds-exporter.policy.json
+
+    # Create IAM policy
+    aws iam create-policy --policy-name ${IAM_POLICY_NAME} --policy-document file:///tmp/prometheus-rds-exporter.policy.json
     ```
 
-2. Start application
+1. Attach IAM role to your EKS cluster
+
+    eksctl will create an IAM role and a Kubernetes service account in Kubernetes
 
     ```bash
-    prometheus-rds-exporter
+    IAM_ROLE_NAME=prometheus-rds-exporter
+    EKS_CLUSTER_NAME=default # Replace with your EKS cluster name
+    KUBERNETES_NAMESPACE=default # Replace with the namespace that you want to use
```

Comment:
To avoid any ambiguity should we precise that's should be the same namespace than the one we'll use to deploy the service with helm instead of `Replace with the namespace that you want to use` ? 🙂

Reply:
As discussed, I updated it